packages/plugins/rubygem-foreman_templates/rubygem-foreman_templates.spec

```diff
@@ -139,6 +139,9 @@ cp -pa .%{gem_dir}/* \
 exit 0
```

Comment:
I think this section will be an issue since the `foreman_restart` macro was dropped by @ehelms